app/components/billing-resubscribe-button.js

```diff
@@ -1,7 +0,0 @@
-import Component from '@ember/component';
-import { reads, and } from '@ember/object/computed';
-
-export default Component.extend({
-  hasSubscriptionPermissions: reads('account.hasSubscriptionPermissions'),
-  canResubscribe: and('subscription.isResubscribable', 'hasSubscriptionPermissions'),
-});
```

Comment:
Are these changes intentional? They seem quite irrelevant, so wanted to bring it to your attention.

Reply:
Ah, gotcha 👍 Then it's good to go 🚀